crates/driver/src/infra/api/routes/quote/dto/quote.rs

```diff
@@ -36,6 +37,8 @@ pub struct Quote {
     solver: eth::H160,
     #[serde(skip_serializing_if = "Option::is_none")]
     gas: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    tx_origin: Option<eth::H160>,
```

Comment:
Do you mean enable the connected solver to tell the driver which `tx_origin` is required?
I expect the required `tx_origin` does not need to change for individual requests and I'd like to go for the simpler solution until there is actually demand for the more sophisticated one.
Also note that this would introduce another difference between `/solve` requests handled for auctions vs. quotes.